src/main/java/codechicken/nei/NEIClientConfig.java

```diff
@@ -279,10 +279,16 @@ public boolean onClick(int button) {
         API.addOption(new OptionToggleButton("inventory.centerSearchWidget", true));
 
         tag.getTag("inventory.focusSearchWidgetOnOpen")
-                .setComment("Focus Search Widget on Open, blurs on mouse move unless typing has started first")
+                .setComment(
+                        "Focus Search Widget on Open, blurs/unfocuses on mouse move unless typing has started first")
                 .getBooleanValue(false);
         API.addOption(new OptionToggleButton("inventory.focusSearchWidgetOnOpen", true));
 
+        tag.getTag("inventory.firstInvCloseClosesInSearch").setComment(
+                "Pressing the open inventory key when the inventory\nwas just opened when the search is focused will\nclose it instead of typing in the search\n(This will prevent typing the inventory open\nkey into the search as the first input though)")
```

Comment:
please write a short comment

Reply:
Shortened, is this good or do you have a different wording suggestion?